README.md

```diff
@@ -1,2 +1,55 @@
-# supervision-success-component
+# Supervision Success
+
 A React component powering a miniature web app for exploring the impact of success in supervision on the justice system at large
+
+## Contents
+
+1. [Input File Format](#input-file-format)
+
+## Input File Format
+
+Example of correct CSV could be found in [`params.csv`](/src/params.csv) file.
```

Comment:
Sorry in advance but I'm going to make a bunch of grammatical comments! ❤️ 

Let's reword this to, "An example of a valid CSV file can be found in ..."

Reply:
@jessex No problem! Thanks for the help with this!